lib/app/item.ex

```diff
@@ -6,18 +6,18 @@ defmodule App.Item do
   alias __MODULE__
 
   schema "items" do
-    field :person_id, :id
-    field :status_code, :integer
     field :text, :string
 
+    belongs_to :status, App.Status, on_replace: :nilify
```

Comment:
From a beginner's perspective, it doesn't make _any_ sense that an `item` would "belongs_to" a Status. 
An `item` can _have_ a `status` but it doesn't "belong" to the status. 
This is immediately confusing and adds no value to the MVP.